src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/FrameworkReference.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="FrameworkReference"
```

Comment:
New rule files need to be listed in CommonFiles.swr too

Reply:
Reviewing #4883 is on my TODO list, which will cover this.